flask_security/forms.py

```diff
@@ -161,8 +161,9 @@ def __init__(self, *args, **kwargs):
         if request.method == 'GET':
             self.email.data = request.args.get('email', None)
 
-    def validate(self):
-        if not super(SendConfirmationForm, self).validate():
+    def validate(self, extra_validators=None):
+        if not super(SendConfirmationForm, self)\
```

Comment:
Very minor:
```suggestion
        if not super()\
```
and others :)